src/styles.scss

```diff
@@ -123,19 +129,43 @@ ul.square-list li {
     font-weight: 400;
 }
 
+.mat-raised-button:hover {
+    background-color: mat-color($rmm-default-highlight);
+}
```

Comment:
We should also set the foreground colour here: otherwise, (maybe only on a dark system theme?) the background becomes white and the text colour also becomes white:

![hover](https://user-images.githubusercontent.com/86378/118635564-05038a80-b7d4-11eb-8ee7-4cd79c277241.png)

Reply:
This should now be fixed by reverting this style and instead using the `color` parameter that is used elsewhere in the app.

It's possible all raised buttons should have the primary color but there might ones that should be excluded, so I refrained from adding a global style at this point.